test/fixtures/templater/jinja_l_metas/002.yml

```diff
@@ -130,11 +124,12 @@ file:
           - dedent: ''
         - newline: "\n"
         - whitespace: '        '
-        - placeholder: '{% if loop.first %}{% else %}{% endif %}'
         - newline: "\n"
         - whitespace: '    '
         - newline: "\n"
+        - placeholder: '{% if loop.first %}'
         - whitespace: '        '
+        - placeholder: "\n            {{action}}_raw_effect_sizes\n        {% else %}"
```

Comment:
Looks a little off, but kind of a side effect of this fix, and no real impact to the overall SQL so fine with this.

Reply:
In a perfect world, what would you hope to see here? I'm guessing the literal and the `{% else %} as two separate placeholders?

Reply:
Well previously we never saw the literal (`{{action}}_raw_effect_sizes\n`) for non-first loops and it was just blank. But it's a placeholder, and it's not incorrect to have this value here, so not really bothered about this change.

Reply:
Reverted so resolving.